doc/deprecation_process.md

```diff
@@ -2,57 +2,169 @@ This page can be linked using: [aka.ms/azsdk/python/deprecation-process](https:/
+  > ## 1.2.4 (2023-03-31)
+  >
+  > ### Other Changes
+  >
+  > - This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
+  > - For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
 
+## sdk_packaging.toml
+
+- Add `auto_update = false` if not already present to avoid the bot overriding your changes.
+
+## pyproject.toml
+
+- Ensure `ci_enabled = false` is NOT present in pyproject.toml. If it is, remove the line, as this will prevent you from releasing the package.
+
+## setup.py
+
+- Update the `Development Status` classifier in `setup.py` to `Development Status :: 7 - Inactive`.
+  - `Inactive` packages are disabled from most CI verification such as tests/mypy/pylint/etc., therefore the CI should be faster and have fewer requirements.
+
+# Step 2: Resolve all open issues/PRs corresponding to the library.
+
+If there is a replacement library, provide a link to the new library or an existing migration guide before closing.
+
+# Step 3: Create a PR
+
+Create a PR targeting the `main` branch.
+
+Example PR to deprecate azure-cognitiveservices-language-luis [here.](https://github.com/Azure/azure-sdk-for-python/pull/36893/files)
+
+## Fix any CI issues
+
+Wait for the CI to run. Fix any issues related to deprecation in the PR, such as CHANGELOG.md or README.md formatting.
+
+There should not be any tests or mypy/pylint/etc. failures as these checks are disabled on `Inactive` packages.
+
+## Post your PR in the Python review channel
+
+Post your PR in the [review channel for Python](https://teams.microsoft.com/l/channel/19%3a4175567f1e154a80ab5b88cbd22ea92f%40thread.skype/Language%2520-%2520Python%2520-%2520Reviews?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for verification that all requirements for deprecation have been met. If you are not the codeowner, please explicitly tag the codeowner in the post for approval.
+
+## Merge PR
+
+Once the PR is approved, merge.
+
+# Step 4: Trigger a release 
+
+A release here is the same as usual, triggering the release pipeline of your SDK. More instructions on release can be found at: https://aka.ms/azsdk/release-checklist
+
+**Note: This release DOES NOT need to be done during during release week and can be done any time.**
+
+## Post-Release
+
+Check to make sure that the new version of the package has been released on PyPI.
+
+# Step 5: Create a new PR to remove package from CI
+
+- Remove the package artifact from `mypackage/ci.yml`. More specifically, remove the `name` and corresponding `safeName` lines from under `Artifacts` here:
+
+```
+extends:
+  parameters:
+    Artifacts:
+    - name: azure-mypackage    
+      safeName: azuremypackage
```

Comment:
In this example, if we remove name and safeName, I think we'd get an invalid yaml. Should the entire ci.yml just be deleted in this case?

Reply:
good point! I would also say we should just delete artifacts since we keep the rest of the files. So it feels inconsistent to delete only the ci.yml. Thoughts?

Reply:
yeah delete artifacts seems good if there is only one package